hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MultiVersionConcurrencyControl.java

```diff
@@ -225,11 +233,12 @@ void waitForRead(WriteEntry e) {
     synchronized (readWaiters) {
       while (readPoint.get() < e.getWriteNumber()) {
         if (count % 100 == 0 && count > 0) {
-          LOG.warn("STUCK: " + this);
+          long totalWaitTillNow = READPOINT_ADVANCE_WAIT_TIME * count;
+          LOG.warn("STUCK for : " + totalWaitTillNow + " millis. " + this);
```

Comment:
Shall we use para{} log style?

Reply:
Ideally WARN level will be enabled always. That is why I thought its ok to keep this existing way. I pushed master branch already this way. If u strongly feel we need to change,  I can change. Might have to put addendum on Master. wdyt?

Reply:
ok, it's fine

Reply:
I mean it's ok to keep the same, since it is already in master.